utils/appliance/__init__.py

```diff
@@ -1893,6 +1894,113 @@ def reset_automate_model(self):
         with self.ssh_client as ssh_client:
             ssh_client.run_rake_command("evm:automate:reset")
 
+    def clean_appliance(self):
+        starttime = time()
+        self.ssh_client.run_command('service evmserverd stop')
+        self.ssh_client.run_command('sync; sync; echo 3 > /proc/sys/vm/drop_caches')
+        self.ssh_client.run_command('service collectd stop')
+        self.ssh_client.run_command('service rh-postgresql95-postgresql restart')
```

Comment:
It might not always be `rh-postgresql95` unfortunately. Use `.format(self.db.postgres_version)` instead.